### PR TITLE
Remove non-ASCII characters and trailing whitespace

### PR DESCRIPTION
**Pull request by keithc-ca**

### CODE REVIEW

**Comment by keithc-ca** [comment]
`longabout.html` didn't have a copyright notice itself before: should we add one?

File: longabout.html [hunk 1]
```diff
@@ -8,41 +8,41 @@
 <body lang="EN-US">
 <h2>About This Content</h2>
 
-<p><em>January 24, 2018</em></p>
+<p><em>October 2, 2019</em></p>
```
Comment (pshipton) [inline longabout.html]: The terms haven't changed, so the date shouldn't either.

File: runtime/compiler/z/codegen/J9TreeEvaluator.cpp [hunk 2]
```diff
@@ -764,9 +764,9 @@ TR::Register * caseConversionHelper(TR::Node* node, TR::CodeGenerator* cg, bool
 
    // Characters a-z (0x61-0x7A) when to upper and A-Z (0x41-0x5A) when to lower
    generateVRIaInstruction (cg, TR::InstOpCode::VLEIH, node, alphaRangeVector, isToUpper ? 0x617A : 0x415A, 0x0);
-   // Characters àáâãäåæçèéêëìíîïðñòóôõö (0xE0-0xF6) when to upper and ÀÁÂÃÄÅÆÇÈÉÊËÌÍÎÏÐÑÒÓÔÕÖ (0xC0-0xD6) when to lower
+   // Characters (0xE0-0xF6) when to upper and (0xC0-0xD6) when to lower
```
Comment (pshipton) [inline runtime/compiler/z/codegen/J9TreeEvaluator.cpp]: @fjeremic are you ok with this change?

**Comment by pshipton** [comment]
> longabout.html didn't have a copyright notice itself before: should we add one?

My vote is to leave it as-is. The file contains licenses which I don't think the standard copyright would cover.

File: runtime/compiler/z/codegen/J9TreeEvaluator.cpp (hunk 2 above)
Comment (fjeremic) [reply runtime/compiler/z/codegen/J9TreeEvaluator.cpp]: This is fine.

**Comment by keithc-ca** [comment]
Reverted date change in `longabout.html`.

File: longabout.html (hunk 1 above)
Comment (keithc-ca) [reply longabout.html]: Reverted.

**Comment by pshipton** [comment]
jenkins test sanity,extended zlinux jdk8

**Comment by pshipton** [comment]
@keithc-ca there is a merge conflict that needs to be resolved.

**Comment by keithc-ca** [comment]
Fixed merge conflict and rebased.

**Comment by pshipton** [comment]
The previous PR build passed, and the travis build succeeded, this is ready to merge.
https://ci.eclipse.org/openj9/job/PullRequest-OpenJ9/1404/

**Pull request merged**